src/components/QuickGrid/DataSelectors.ts

```diff
@@ -24,22 +22,27 @@ const sortRows = (rows: Array<any>, sortColumn: string, sortDirection: SortDirec
     }
 
     if (groupedColumn.length > 0) {
-        const groupSortDir: string = groupBySortDirection === SortDirection.Descending ? 'desc' : 'asc';
-        const groupSortBy = (groupBySortColumn !== undefined && groupBySortColumn !== '') ? groupBySortColumn : groupedColumn[0];
+        let sortColumns = [];
+        let sortDirections = [];
+        for (let groupColumn of groupedColumn) {
```

Comment:
Added the use of custom sort function on grouped columns, if the function is defined